.ruby-version

```diff
@@ -1 +1 @@
-2.7
```

Comment:
I wasn't sure if I should change this, but I didn't see a test version matrix anywhere so bumped this up.

Reply:
I should probably start switching everything over, so 👍